### DISCUSSION TITLE
Internet speed - minimum recommended requirements for Codespaces

### DISCUSSION THREAD

**Discussion by Maxhodges** [community/community#29399] [Codespaces]
i'm concerned about Codespaces developer experience. Do you have any minimum recommended internet speed requirements for Codespaces? We don't really want to spend the time migrating our monorepo to Codespaces only to find out our engineer in Africa will suffer terrible latency. Any guidance to help us would be appreciated.

**Comment by MadhushaPrasad** [comment]
You can check this link to get some details about [minimum specification for codespace machines](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/setting-a-minimum-specification-for-codespace-machines)
